docker/build-tools/Dockerfile

```diff
@@ -904,6 +905,10 @@ RUN rm -fr /usr/share/locale
 RUN rm -fr /usr/share/man
 RUN rm -fr /tmp/*
 
+# Install cargo-deny
```

Comment:
I think we want this at L517. The dockerfile is kind of complex but we actually build 2 images from 1 Dockerfile, so this is putting it in the wrong final image.